packages/pds/src/db/tables/app-password.ts

```diff
@@ -0,0 +1,10 @@
+export interface AppPassword {
+  accountDid: string
```

Comment:
No issue here at all, but I'm interested to hear whether there's any special reason this is `accountDid` rather than just `did`, as you might find in some of the other tables.

Reply:
no good reason lol. i'll update it 👍